networks.json

```diff
@@ -1,316 +1,342 @@
-      "cover": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/covers/band-protocol.png?sanitize=true",
-      "links": [
-          {
-              "name": "Retired",
-              "chain_id": "band-guanyu-mainnet",
-              "url": "https://band-guanyu.bigdipper.live"
-          },
-          {
-              "name": "Mainnet",
-              "chain_id": "laozi-mainnet",
-              "url": "https://band.bigdipper.live"
-          }
-      ]
+    "name": "Band Protocol",
+    "endpoint": "https://gql.band.forbole.com/v1/graphql",
+    "links": [
+      {
+        "name": "Retired",
+        "chain_id": "band-guanyu-mainnet",
+        "url": "https://band-guanyu.bigdipper.live"
+      },
+      {
+        "name": "Mainnet",
+        "chain_id": "laozi-mainnet",
+        "url": "https://band.bigdipper.live"
+      }
+    ]
   },
   {
-      "name": "LikeCoin",
-      "logo": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/logos/likecoin.svg?sanitize=true",
-      "cover": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/covers/likecoin.png?sanitize=true",
-      "links": [
-          {
-              "name": "Mainnet",
-              "chain_id": "likecoin-mainnet-2",
-              "url": "https://bigdipper.live/likecoin"
-          },
-          {
-              "name": "Testnet",
-              "chain_id": "likecoin-public-testnet-5",
-              "url": "https://testnet.likecoin.bigdipper.live"
-          }
-      ]
+    "name": "LikeCoin",
+    "endpoint": "https://gql.likecoin.forbole.com/v1/graphql",
+    "links": [
+      {
+        "name": "Mainnet",
+        "chain_id": "likecoin-mainnet-2",
+        "url": "https://bigdipper.live/likecoin"
+      },
+      {
+        "name": "Testnet",
+        "chain_id": "likecoin-public-testnet-5",
+        "url": "https://testnet.likecoin.bigdipper.live"
+      }
+    ]
   },
   {
-      "name": "Desmos",
-      "logo": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/logos/desmos.svg?sanitize=true",
-      "cover": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/covers/desmos.png?sanitize=true",
-      "links": [
-          {
-              "name": "Testnet",
-              "chain_id": "morpheus-apollo-2",
-              "url": "https://morpheus.desmos.network"
-          },
-          {
-              "name": "Mainnet",
-              "chain_id": "desmos-mainnet",
-              "url": "https://explorer.desmos.network"
-          }
-      ]
+    "name": "Desmos",
+    "endpoint": "https://gql.desmos.forbole.com/v1/graphql",
+    "links": [
+      {
+        "name": "Testnet",
+        "chain_id": "morpheus-apollo-2",
+        "url": "https://morpheus.desmos.network"
+      },
+      {
+        "name": "Mainnet",
+        "chain_id": "desmos-mainnet",
+        "url": "https://explorer.desmos.network"
+      }
+    ]
```

Comment:
I believe this should be first `Mainnet` and then `Testnet` to make sure the order is the same as other chains, am I wrong?